extensions/elasticsearch-rest-client-common/runtime/pom.xml

```diff
@@ -46,8 +46,8 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.graalvm.nativeimage</groupId>
-            <artifactId>svm</artifactId>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>graal-sdk</artifactId>
             <scope>provided</scope>
```

Comment:
Here too and in a lot of other places below.

Let's decide if we make it provided or not and make sure it's consistent everywhere.

Reply:
@gsmet https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-scope suggests it should be `provided` everywhere, since the runtime use of this dependency is native compilation. And GraalVM would have that dependency available. Do you agree?

Reply:
`provided` would have my preference but... I'm wondering if we are still using `ImageInfo` from the `graal-sdk` jar. If so, we could change them to use `io.quarkus.bootstrap.graal.ImageInfo`, which would be better and make everything `provided`.

Reply:
Thanks! Looks like quarkus runtime has some `ImageInfo` usage from Graal still:

```
core/runtime/src/main/java/io/quarkus/runtime/ImageMode.java:3:import org.graalvm.nativeimage.ImageInfo;
core/runtime/src/main/java/io/quarkus/runtime/ImageMode.java:40:        if (ImageInfo.inImageBuildtimeCode()) {
core/runtime/src/main/java/io/quarkus/runtime/ImageMode.java:42:        } else if (ImageInfo.inImageRuntimeCode()) {
```

This should fail in tests, right? I'll make it provided and see it going red. So we might need another fix before we can fully transition.

Reply:
They all need to be replaced with `io.quarkus.bootstrap.graal.ImageInfo`.

Reply:
Shall I update it in this PR or a separate one?

Reply:
Let's do this in this PR.